app/src/main/java/com/hedvig/app/feature/offer/OfferViewModel.kt

```diff
@@ -92,10 +94,11 @@ class OfferViewModelImpl(
                 ViewState.HasContracts
             } else {
                 val topOfferItems = OfferItemsBuilder.createTopOfferItems(data)
-                val documentItems = OfferItemsBuilder.createDocumentItems(data.quoteBundle.quotes[0])
+                val perilItems = OfferItemsBuilder.createPerilItems(data.quoteBundle.quotes[0])
                 val insurableLimitsItems = OfferItemsBuilder.createInsurableLimits(data.quoteBundle.quotes[0])
+                val documentItems = OfferItemsBuilder.createDocumentItems(data.quoteBundle.quotes[0])
                 val bottomOfferItems = OfferItemsBuilder.createBottomOfferItems(data)
-                ViewState.OfferItems(topOfferItems, documentItems, insurableLimitsItems, bottomOfferItems)
+                ViewState.OfferItems(topOfferItems, perilItems, documentItems, insurableLimitsItems, bottomOfferItems)
```

Comment:
Lots of parameters here now, we could name them to increase readability.

Reply:
Agreed. I'll address this comment in the branch I'm working on right now to avoid a merge conflict, however. 🙂